ndt-server.go

```diff
@@ -24,21 +22,22 @@ import (
 	"github.com/m-lab/ndt-server/ndt7/spec"
 	"github.com/m-lab/ndt-server/platformx"
 
+	"github.com/justinas/alice"
```

Comment:
Is this middleware chaining library named alice because of Alice in Chains?

Reply:
Hahaha, you must be right. I originally thought it was Alice in Wonderland, but the package literally has  `alice.Chains` data type.